web/flat-web/src/stores/WhiteboardStore.ts

```diff
@@ -79,6 +84,10 @@ export class WhiteboardStore {
         this.isShowPreviewPanel = show;
     };
 
+    public preloadPPTResource = debounce((pptSrc: string): void => {
+        getCoursewarePreloader.preload(pptSrc);
+    }, 2000);
```

Comment:
```ts
// we can't use static here because we want to make sure that the function is up to date every time we instantiate the class
private preloadPPTResource: Promise<void> = debounce(getCoursewarePreloader.preload, 2000);
```